### PR TITLE
Multisig - Add new test-cases for fund transfer Tx

### PR DESCRIPTION
**Pull request by githubckgoh1439**
Hi, YK

Added some test-cases for Create-multisig-tx.

### CODE REVIEW

**Comment by githubckgoh1439** [comment]
Create-multisig-Tx - Issue on Re-submit using unique nonce, TxID


Hi, YK

Found some abnormal situations base on above cases 
(Refer to case-3 inside test-0700-multisig.ts).

Attached with the examples of DB state query :

```
Case-1. While query base on group-address, TWO pendingTX found. But I expected is THREE pendingTx. Using the same copy of code.

Case-2. While query base on group-address, only ONE pendingTX found. But I expected is THREE pendingTx. Using the same copy of code.
```
[[postman-sample-1]_get_multisig_acc.txt](https://github.com/yenkhoon/mxw-sdk-js/files/4414870/postman-sample-1._get_multisig_acc.txt)
[[postman-sample-2]_get_multisig_acc.txt](https://github.com/yenkhoon/mxw-sdk-js/files/4414871/postman-sample-2._get_multisig_acc.txt)



Please check and update.


TQ.

**Comment by yenkhoon** [comment]
how many internal transaction u created?

File: src.ts/wallet.ts [hunk 1]
```diff
@@ -139,9 +139,22 @@ export class Wallet extends AbstractSigner {
         }
         return resolveProperties(transaction).then((tx) => {
             console.log("txtxtxtxxtxtxtxtxtxtx" + JSON.stringify(tx));
-            if (!tx.nonce || !tx.accountNumber || !tx.value || !tx.value.msg || !Array.isArray(tx.value.msg)) {
+            if (!tx.nonce) {
                 errors.throwError('missing transaction field', errors.MISSING_ARGUMENT, { argument: 'value', value: tx });
```
Comment (githubckgoh1439) [inline src.ts/wallet.ts]: Hi, YK

The debugger show the Error in here, while come to the process signed by signer-2.

Attached with the compare between signer-1 & signer-2 for their InternalTx signing. 


Please check.

[Tx_by_signer2.txt](https://github.com/yenkhoon/mxw-sdk-js/files/4433763/Tx_by_signer2.txt)

[Tx_by_signer1.txt](https://github.com/yenkhoon/mxw-sdk-js/files/4433764/Tx_by_signer1.txt)

Comment (githubckgoh1439) [reply src.ts/wallet.ts]: This close.

**Comment by githubckgoh1439** [comment]
Resolved the conflict in multisig.ts

**Comment by githubckgoh1439** [comment]
Dear all,

Pleace refer to latest of this : 
https://github.com/yenkhoon/mxw-sdk-js/pull/4

This pull request can be ignored.

**Pull request closed without merging**